reps/2022-04-07-serialization.md

```diff
@@ -0,0 +1,162 @@
+## Summary
+### General Motivation
+Current Ray's serialization has some issues:
+
+1. Doesn't support out-of-band serialization in Java worker. Cause some perfomance issues in Java worker.
```

Comment:
define out-of-band

Reply:
@lixin-wei you can add some concrete examples. For example, there was a requirement for zero-copy reading Arrow data in Java.

Reply:
Also, you can talk about how Python currently supports OOB with Pickle. You can link this [PEP](https://peps.python.org/pep-0574/).